lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -3460,6 +3465,37 @@ SPIRVValue *LLVMToSPIRVBase::transIntrinsicInst(IntrinsicInst *II,
   return nullptr;
 }
 
+SPIRVValue *LLVMToSPIRVBase::transFenceInst(FenceInst *FI,
+                                            SPIRVBasicBlock *BB) {
+  SPIRVWord MemorySemantics = static_cast<SPIRVWord>(FI->getOrdering());
+  // Fence ordering may only be Acquire, Release, AcquireRelease, or
+  // SequentiallyConsistent
+  switch (MemorySemantics) {
+  case 4:
+    MemorySemantics = 0x2;
```

Comment:
```suggestion
    MemorySemantics = MemorySemanticsAcquireMask;
```

Same for the hardcoded constants below.

Reply:
Thanks! Changed in [4ff8796](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1390/commits/4ff879629a5bba97459cd450964789570ef5e5b3)